### PR TITLE
Querying lotame data

### PR DESCRIPTION
**Pull request by jeteve**
Added lotame in appnexus and openx for Ozone according to https://trello.com/c/uRWUiBRX

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/prebid/types.js [hunk 1]
```diff
@@ -36,11 +36,13 @@ export type PrebidXaxisParams = {
 export type PrebidAppNexusParams = {
     placementId: string,
     customData: string,
+    lotame?: {},
```
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/prebid/types.js]: Why will this be a maybe? We seem to always add it. I think the value could sometimes be undefined, if the `window. OzoneLotameData` is missing...

Comment (jeteve) [reply static/src/javascripts/projects/commercial/modules/prebid/types.js]: We don't add it for appnexus direct for instance. I'll cover the undefined case :) Thanks!

Comment (ScottPainterGNM) [reply static/src/javascripts/projects/commercial/modules/prebid/types.js]: That's a good point actually, @jeteve are you happy with what should have lotame and what shouldn't? 

Happy to run though!

Comment (jeteve) [reply static/src/javascripts/projects/commercial/modules/prebid/types.js]: Sure @ScottPainterGNM . As far as I know, this lotame account we have will be used by Ozone for audience management (the account will be shared across partners). So it should be only what goes to Ozone.

**Pull request merged**